14:53:10 139.102.14.201 14:53:10 Mon Apr 28, 2014
14:53:30 139.102.14.201 14:53:30 Mon Apr 28, 2014
14:53:43 139.102.14.201 14:53:43 Mon Apr 28, 2014
15:54:35 10.139.60.243 15:54:35 Mon Apr 28, 2014
16:16:19 139.102.14.201 16:16:19 Mon Apr 28, 2014
16:16:50 139.102.14.201 16:16:50 Mon Apr 28, 2014
16:17:00 139.102.14.201 16:17:00 Mon Apr 28, 2014
16:17:37 139.102.14.201 16:17:37 Mon Apr 28, 2014
Traceback (most recent call last):
  File "./server.py", line 18, in <module>
    desc.send(msg)
socket.error: [Errno 104] Connection reset by peer
